app/views/pregnancies/_notes.html.erb

```diff
@@ -1,8 +1,33 @@
 <section id="notes" class="notes tab-pane">
 	<div class="margin-bottom col-md-12">
 		<h1 class="border-bottom title">Notes</h1>
-    <div class="info-form-left col-md-6" id="notes_content">
-      Something here.
-    </div> 
+        <div class="info-form-left col-md-6">
+					<%= bootstrap_form_for [@pregnancy, note] do |f| %>
+				    <%= f.text_area :full_text, placeholder: 'Enter notes here', hide_label: true %>
+				    <%= f.submit class: "btn btn-primary"  %>
+				  <% end %>
+        </div>
 	</div>
+
+	<table class="table border-side">
+		<tbody>
+		<% notes.order('created_at DESC').each do |note| %>
+		<tr>
+				<td class="col-md-2 bold">
```

Comment:
Ah, I see -- you're declaring a css class to bold the inner text here. I think that using a `th` instead of a `td` will achieve pretty much the same effect without the need for additional css, let me take a stab.